src/Medusa.ts

```diff
@@ -28,69 +28,67 @@
  * Load medusa and apply all components
  */
 export class Medusa {
-	readonly #express: Express;
-	readonly #rootDir: string;
+    readonly #express: Express;
```

Comment:
Codacy has a fix for the issue: [Replace `····` with `↹`](https://app.codacy.com/gh/adrien2p/medusa-extender/pullRequest?prid=9061797)
```suggestion
	readonly #express: Express;
```